docs/configuration.md

```diff
@@ -90,7 +90,7 @@ val sc = new SparkContext(new SparkConf())
 Then, you can supply configuration values at runtime:
 {% highlight bash %}
 ./bin/spark-submit --name "My app" --master local[4] --conf spark.eventLog.enabled=false
-  --conf "spark.executor.extraJavaOptions=-XX:+PrintGCDetails -XX:+PrintGCTimeStamps" myApp.jar
+  --conf "spark.driver.extraJavaOptions=-XX:+PrintGCDetails -XX:+PrintGCTimeStamps" myApp.jar
```

Comment:
Hi, @FereX98 . So, does this PR aims to match this with `--master local[4]`?

Reply:
Yeah I guess so

Reply:
Could you check the other example in this file?
For example, line 2725 seems to be the same with this example.
- https://github.com/apache/spark/blame/19ffc9096339bde87f774b92f16c2441adf5841c/docs/configuration.md#L2725

Reply:
This is fine. It's also fine to say `--master your-master-here` or something to indicate this doesn't matter